src/backend/distributed/shared_library_init.c

```diff
@@ -105,9 +106,6 @@
 
 #include "columnar/columnar.h"
 
-/* marks shared object as one loadable by the postgres version compiled against */
-PG_MODULE_MAGIC;
```

Comment:
Why remove this?

Reply:
It was removed because we were getting 'Redefinition error'. We are introducing decoder plugin which needs this declaration.

Reply:
let's keep this here